deploy/olm-catalog/unifiedpush-operator/unifiedpush-operator.package.yaml

```diff
@@ -0,0 +1,4 @@
+packageName: integreatly-unifiedpush
+channels:
+- currentCSV: unifiedpush-operator.v0.2.0
+  name: integreatly
```

Comment:
I'm not exactly sure what channels do. Is integreatly the only one we want to include? Does that limit the use of the operator on OS4 upstream?

Reply:
My understand are channels just points to specified csvs, to allow versions/upgrades depending on stability etc and it's possible to have multiple channels. So i don't think it should limit the use of the operator on OS4 upstream. Again @matskiv may be able to better comment on this, since I'm still quite new to this

Reply:
@KevFan is right, it shouldn't limit us.
Also, for upstream we might just use a different organization for application repo, currently for integreatly we use: quay.io/application/integreatly/integreatly-unifiedpush
for upstream this would probably make more sense: quay.io/application/aerogear/unifiedpush
And we would not put "integreatly" channel into that repo.